src/api/json/catalog.json

```
@@ -6179,7 +6188,15 @@
       "name": "Windows Package Manager Locale Manifest",
       "description": "Windows Package Manager Locale Manifest file, used for detailing locale specific metadata",
       "fileMatch": [
-        "**/manifests/?/*/*/*/*.*.locale.{en-US,fr-FR,it-IT,ja-JP,ko-KR,pt-BR,ru-RU,zh-CN,zh-TW}.yaml"
+        "**/manifests/?/*/*/*/*.*.locale.en-US.yaml",
+        "**/manifests/?/*/*/*/*.*.locale.fr-FR.yaml",
+        "**/manifests/?/*/*/*/*.*.locale.it-IT.yaml",
+        "**/manifests/?/*/*/*/*.*.locale.ja-JP.yaml",
+        "**/manifests/?/*/*/*/*.*.locale.ko-KR.yaml",
+        "**/manifests/?/*/*/*/*.*.locale.pt-BR.yaml",
+        "**/manifests/?/*/*/*/*.*.locale.ru-RU.yaml",
+        "**/manifests/?/*/*/*/*.*.locale.zh-CN.yaml",
+        "**/manifests/?/*/*/*/*.*.locale.zh-TW.yaml"
```

I don't have domain knowledge on this one, but I wonder if this needs to be such a specific enumeration, or if it would be better written as just

```suggestion
        "**/manifests/?/*/*/*/*.*.locale.*-*.yaml"
```

(...or `...locale.*.yaml`).

---

That's definitely more appealing as it covers more cases.- Good to leave it as is though